### PR TITLE
Updated WiseApp as per changes in the Workflow api.

### PR DESCRIPTION
**Pull request by sagarkapare**

### CODE REVIEW

File: Wise/README.rst [hunk 1]
```diff
@@ -465,22 +465,23 @@ Each generic parameter of the ``Mapper`` and the ``Reducer`` contains:
 
 Scheduling the MapReduce Job
 ----------------------------
-To schedule the ``BounceCountsMapReduce`` job to run every ten minute, we define it in the
-``WiseWorkflow``::
+To schedule the ``BounceCountsMapReduce`` to run every ten minutes, we need to add it in the
```
Comment (awholegunch) [inline Wise/README.rst]: we need to add it to the

File: Wise/README.rst [hunk 2]
```diff
@@ -465,22 +465,23 @@ Each generic parameter of the ``Mapper`` and the ``Reducer`` contains:
...
-                                  Schedule.Action.START))
-        .build();
+    public void configure() {
+        setName("WiseWorkflow");
+        setDescription("Wise Workflow");
+        addMapReduce("BounceCountsMapReduce");
     }
   }
 
+The ``WiseWorkflow`` then can be scheduled in the ``WiseApp``::
```
Comment (awholegunch) [inline Wise/README.rst]: can then be

File: Wise/src/main/java/co/cask/cdap/apps/wise/WiseApp.java [hunk 3]
```diff
@@ -34,9 +35,13 @@ public void configure() {
     createDataset("bounceCountStore", BounceCountStore.class);
     // Add the WiseFlow flow
     addFlow(new WiseFlow());
+    // Add MapReduce that computes the bounce counts of visited pages
+    addMapReduce(new BounceCountsMapReduce());
     // Add the WiseWorkflow workflow
     addWorkflow(new WiseWorkflow());
     // Add the WiseService service
     addService(new WiseService());
+    // Schedule the Workflow that runs MapReduce program which computes the bounce counts of visited pages
```
Comment (awholegunch) [inline Wise/src/main/java/co/cask/cdap/apps/wise/WiseApp.java]: // Schedule the Workflow to run the MapReduce program every ten minutes

**Comment by awholegunch** [comment]
A few comments, otherwise LGTM.

**Comment by sagarkapare** [comment]
Addressed the comments.

**Pull request merged**